src/doc/blend_funcs.cpp

```diff
@@ -397,6 +397,43 @@ static void set_lum(double& r, double& g, double& b, double l)
   clip_color(r, g, b);
 }
 
+static inline uint8_t get_imin_channel(double& r, double& g, double& b)
```

Comment:
It looks like this function doesn't require references:
```suggestion
static inline uint8_t get_imin_channel(const double r, const double g, const double b)
```